src/shared/rules_op.c

```diff
@@ -67,7 +67,7 @@ int OS_ReadXMLRules(const char *rulefile,
     const char *xml_user = "user";
     const char *xml_url = "url";
     const char *xml_id = "id";
-    const char *xml_data = "extra_data";
```

Comment:
Renaming the option `extra_data` to `data` will produce changes in the parser (user side).
Please undo this change, and refactor `xml_data` to `xml_extra_data`.